macro/src/class_info.rs

```diff
@@ -172,6 +172,10 @@ impl ClassInfo {
             | (Privacy::Default, ClassKind::Class) => false,
         }
     }
+
+    pub fn generics_scope(&self) -> GenericsScope {
```

Comment:
```suggestion
    pub fn generics_scope(&self) -> GenericsScope<'_> {
```

This is the case I find most important! Got to signal that the return is tied to `self`